migrations/20240501024340_vnft_into_ud_tbl.sql

```diff
@@ -0,0 +1,109 @@
+-- +goose Up
```

Comment:
@JamesReate How scary does this migration look to you? Do you think Admin will throw a fit?